packages/docs-site/src/content/docs/network-reference/mainnet-addresses.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Mainnet Addresses
+description: Network reference page describing various important addresses on Taiko.
+---
+
+## Ethereum contracts
+
+| Contract Name (Shared) | Address                                      |
```

Comment:
should we add an ENS column? See:
https://github.com/taikoxyz/taiko-mono/blob/main/packages/protocol/deployments/mainnet-contract-logs-L1.md

Reply:
added